src/validator/CreditCardValidatorProject.java

```diff
@@ -0,0 +1,60 @@
+package validator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+public class CreditCardValidatorProject {
+	
+	private static CreditCardVendorsReader reader = new CreditCardVendorsReader();
+	private static ArrayList<CreditCardVendor> vendors = reader.returnVendorArray();
+	private static CreditCardValidator validator = new CreditCardValidator();
+	private static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+	
+	public static void main(String[] args) {
+        System.out.println("CREDIT CARD VALIDATOR\n");
+        
+        int vendorsLength = vendors.size();
```

Comment:
you don't need an extra variable for that, you can inline this one, it will be easy to understand